### ISSUE TITLE
[Bug] Splatoon 3: Side Order DLC causing Steamdeck to reboot

### ISSUE THREAD

**Issue by alexmtc765** [Ryujinx/Ryujinx#6496]
### Description of the issue

Walking into the tower in the Splatoon 3 Side order DLC causes my Steamdeck to reboot. (Maybe the GPU driver is crashing?)

### Reproduction steps

1. Load into Splatoon 3
2. Load into the Side Order DLC
3. Walk into the Tower
4. Wait on the loading screen

### Log file

[Ryujinx_1.1.1231_2024-03-14_21-13-13.log](https://github.com/Ryujinx/Ryujinx/files/14609499/Ryujinx_1.1.1231_2024-03-14_21-13-13.log)

### OS

SteamOS Holo x86_64

### Ryujinx version

1.1.1231

### Game version

7.0.0

### CPU

AMD Custom APU 0405 (8) @ 2.800GHz 

### GPU

AMD ATI AMD Custom GPU 0405

### RAM

16GB

### List of applied mods

None

### Additional context?

I'm playing on a LCD Steamdeck

Im on a fresh install of SteamOS
Kernel: 6.1.52-valve16-1-neptune-61

I installed Ryujinx with Emudeck
 
After my Steamdeck appears to reboot from the crash, its very unstable with some minor GPU artifacts popping up, these completely go away if I reboot my Steamdeck manually. 
These artifacts only come back when I relaunch Ryujinx and cause the crash to happen again.

I'm on the second tower in the DLC with a save I transferred over to my Steamdeck

I have not been able to find a solution to this issue online

**Comment by piplup55** [comment]
This is a known amd gpu bug it only happens on vulkan i reported it already on the gamelist https://github.com/Ryujinx/Ryujinx-Games-List/issues/4128#issuecomment-1962727343

**Comment by 5at0ri** [comment]
sorry this is a bit unrelated but how did u get the dlc to work in the first place, mine keeps telling me to update the file even tho it already is the latest?

**Comment by CardboardGnat** [comment]
excluding swapping to open GL, unless there is a way to fix the extreme lag spikes and short freezes, have there been any patches for Splatoon 3's DLC on the switch? The spikes on the steam deck make it incredibly difficult to play.

**Comment by riperiperi** [comment]
"swapping to opengl" makes the issue worse, it's Vulkan that has shorter shader compile times.

**Comment by CardboardGnat** [comment]
yeah but the DLC will crash as you enter the "lobby"  whilst under Vulkan, thats why I have to use OpenGL
